tidb-cloud/data-service-postman-integration.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Run Data App in Postman
+summary: Learn how to run your Data App in Postman.
+---
+
+# Run Data App in Postman
+
+[Postman](https://www.postman.com/) is an API platform for building and using APIs, which simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
+
+In TiDB Cloud [Data Service](https://tidbcloud.com/console/data-service), you can easily import your Data App to Postman and leverage Postman's extensive tools to enhance your API development experience.
+
+This document describes how to import your Data App to Postman and how to run your Data App in Postman.
+
+## Before you begin
+
+Before importing a Data App to Postman, make sure that you have the following:
+
+- A [Postman](https://www.postman.com/) account.
+- A [Postman desktop app](https://www.postman.com/downloads) (Optional. You can use the Postman webpage version without downloading the app.)
```

Comment:
A [Postman desktop app](https://www.postman.com/downloads) is available (optional). Alternatively, you can use the Postman web version without downloading the app."

Reply:
```suggestion
- A [Postman desktop app](https://www.postman.com/downloads) (optional). Alternatively, you can use the Postman web version without downloading the app.
```